src/frontend/Komponenter/Behandling/SettPåVent/SettPåVent.tsx

```diff
@@ -85,6 +92,11 @@ type SettPåVentRequest = {
     oppfølgingsoppgaverMotLokalKontor: VurderHenvendelseOppgavetype[];
 };
 
+type IOppgavestatus = {
```

Comment:
Veldig pirk, men tenker denne bare kan hete `OppgaveStatus`. I kommer fra interface, som denne ikke her.

... og det er egentlig heller ingen grunn til å prefixe interfaces med "I", men det er vi flere som surrer 😅  https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120

Reply:
Heller ingen grunn til å prefixe enums med E :)

Reply:
Ja, det er mange "I" prefiks i den frontend sausen. Fjerner denne I'en i alle fall.